framework/doc/content/sqa/framework_sdd.md

```diff
@@ -161,11 +163,10 @@ composed of a single or small set of C++ objects intended to be specialized by a
 specific problem. To accomplish this design goal, MOOSE uses several modern object-oriented design
 patterns. The primary overarching pattern is the "Factory Pattern". Users needing to extend MOOSE
 may inherit from one of MOOSE's systems to providing an implementation meeting his or her needs. The
-design of each of these systems is documented on the mooseframework.org wiki in the Tutorial
-section. Additionally, up-to-date documentation extracted from the source is maintained on the the
-mooseframework.org documentation site after every successful merge to MOOSE's stable branch. After
-these objects are created, the can be registered with the framework and used immediately in a MOOSE
-input file.
+design of each of these systems is documented on the [MOOSE homepage](https://mooseframework.inl.gov).
```

Comment:
Definitely more resilient to changes if I do that - good call.

Reply:
Turns out this would require apps that refer to this section of the framework SDD (several do right now) will fail unless I manually add this file to the available content. Going to stick with the URL, as we don't have plans to change that in the foreseeable future.